dotfiles/xmonad/xmonad.hs

```diff
@@ -183,7 +183,7 @@ main = do
     , workspaces         = myWorkspaces
     , manageHook         = newManageHook
     , handleEventHook    = fullscreenEventHook <+> docksEventHook
-    , startupHook        = setWMName "LG3D" >> spawn "~/essentials/scripts/lemonbar/bar_start.sh"
```

Comment:
We no longer need `bar_start` to launch that little status bar on the right. We properly insert this data into the original bar.